src/HtmlPurifier.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WMDE\Fundraising\HtmlFilter;
+
+use HTMLPurifier_Config;
+use HTMLPurifier as OriginalHTMLPurifier;
+
+class HtmlPurifier implements PurifierInterface {
+
+	/**
+	 * @var OriginalHTMLPurifier
+	 */
+	private $purifier;
+
+	private static $allowedTags = 'p,b,a[href],i,strong,em,span,ul,ol,li,h1,h2,h3,h4,h5,h6,br,img[src|alt]';
```

Comment:
Well. I get what you mean, but the whole point of this repo is to encapsulate this configuration. We could put it into a config file - but we do not want this overwriteable by the user of our class, as far as I am concerned.